### ISSUE TITLE
todo-modern: Accessing PropTypes via the main React package is deprecated.

### ISSUE THREAD

**Issue by joseluisq** [relayjs/relay-examples#20]
When I start via `yarn start` the TodoMVC example at `./todo-modern` then the browser console throws this __warning__:

```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```

**Comment by kassens** [comment]
This is coming directly from Relay. I'm working on it to get it fixed with the next release.

**Comment by joseluisq** [comment]
:+1:

**Comment by pranaygp** [comment]
@joseluisq seems like this has already been fixed (either with 3ffe487674e30a73c603a094129612c11a966ec4 or with a new relay release as @kassens mentioned).

After pulling the latest version of this repo, if this issue still repros for you, please reopen this (doesn't repro for me with the current master)

**Comment by joseluisq** [comment]
@pranaygp Sure, I will try again for run it using the new Relay release.
